test/API/CMake/CheckAsan.cmake

```diff
@@ -0,0 +1,37 @@
+set(ASAN_FLAG "-fsanitize=address")
```

Comment:
Isn't this a duplicate of cmake/sanitizer/sanitizers.cmake?

Reply:
More or less, yes. Asan and Ubsan were used in the external VOL tests repo during testing.